entrypoint.sh

```diff
@@ -46,5 +46,17 @@ echo "=> Updating Youtube-dl..."
 $YTDL_BIN -U
 echo "=> Updated youtube-dl with exit code $?"
 
+echo "=> Checking network connection"
+ping -c1 sinusbot.com
+if [ $? -eq 0 ]
+then
+    echo "=> Updating Sinusbot"
+    cd /sinusbot
+    wget https://www.sinusbot.com/dl/sinusbot.current.tar.bz2
+    tar -xjvf sinusbot.current.tar.bz2
+    cp plugin/libsoundbot_plugin.so TeamSpeak3-Client-linux_amd64/plugins/
+    chown -R sinusbot /sinusbot
+    echo "=> Done"
+fi
```

Comment:
As written please remove this download, it is not good to download something. After your PR I will also move the above `youtube-dl` update process higher and look into having a scheduled build every week or so.

Reply:
Setting up the Docker when there is a new version of sinusbot is annoying for the user, especially when he uses large music libraries. 
Updating at the start as you said is not a good idea.

I removed the code but please provide a script to update the bot inside the docker without doing the resetup.